components/styled/table/src/atoms/table-cell.tsx

```diff
@@ -0,0 +1,42 @@
+import { FC } from 'react'
```

Comment:
This file is very much a WIP - I'm not sure if it's necessary and we may well remove it.
It's not currently used so can remove from this PR if anyone objects, but it makes less sense moving Table Atom -> Molecule and QuestionnaireTable molecule -> organism if this component isn't around

Reply:
I've decided to remove this for now - it doesn't belong in this PR